src/components/BioMose.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import mose from '../../src/img/mose.png'
+
+const BioMose = () => {
+  return (
+    <div className="bg-slate-50">
+      <div className="px-6 md:px-16 lg:px-24 pt-8 md:pt-16">
+        <h2 className="text-5xl md:text-6xl mb-3 text-center">About</h2>
+        <hr className="mx-4 md:mx-20 bg-black" />
+        <p className="py-8 md:py-12 text-lg md:text-xl">
```

Comment:
Based on the figma design the font size should be text-base md:text-2xl. But I could be wrong. Maybe double check with the design on figma?